apps/cli/src/util/pagination-options.ts

```diff
@@ -0,0 +1,32 @@
+import { CommandOption } from '@/types/command/command.types'
+
+export const getPaginationOptions = (): CommandOption[] => {
```

Comment:
I don't think a function is necessary. a `PAGINATION_OPTION` constant would work just fine